package.json

```diff
@@ -43,8 +43,6 @@
     "@types/jest": "^26.0.15",
     "@types/node": "^13.0.0",
     "@types/semver": "^7.3.4",
-    "@types/web3": "1.2.2",
```

Comment:
I think that the tests are failing because of this. Why did we remove it?

Reply:
It should be workin, I test it locally, let me check again.